spec/steps/backend_steps.rb

```diff
@@ -104,6 +104,19 @@
   expect(page).to_not have_content('ACMR5')
 end
 
+step 'I can set the option of hiding the comment field of a resource type' do
+  step 'I click on Resource types menu item'
+  find('tr', text: 'Virtual PC').click_link ('Edit')
+  check 'Hide the comment field'
+  click_button 'Update resource type'
+end
+
+step 'I can see that the new booking pages of this resource type do not show the comment field' do
+  step 'I go to the bookings page'
+  step 'I can start to create a booking'
+  expect(page).to_not have_content('Comment')
```

Comment:
¡Ah, olvidaba!

Ensure the comment field appears before enabling the hide option, to ensure the test fails if for other reason the field doesn't appear here.

Reply:
Ok, I have added some lines to ensure us the comment field is visible before checking the hide option.